### DISCUSSION TITLE
Add users to multiple organizations automatically using GitLab OAuth

### DISCUSSION THREAD

**Discussion by jpweng** [grafana/grafana#57907] [General]
Is it possible to add users to multiple organizations automatically using GitLab OAuth?

Using https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#auto_assign_org_id, a user authenticated using GitLab OAuth can be automatically added to an organization, specified by the organization id. Combined with https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#oauth_skip_org_role_update_sync, such a user can then be manually added to other organizations.

Is it possible to add a user to multiple organizations automatically?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
